### ISSUE TITLE
syntax error: token -> '}'

### ISSUE THREAD

**Issue by Unbedingt** [lronaldo/cpctelera#77]
bash
```
syntax error: token -> '}' ; column 1
make: *** [obj/mapa.rel] Error 1
```

File mapa.c
```
// Generated CSV tilemap from img/mapa.tmx
const u8 mapa[50*25] = {
      why is it empty?
};
```

Info:
Tiled for macOS version 1.1.6

**Comment by Unbedingt** [comment]
Antwort:
Documentation CPCtelera:
This script converts tmx files saved in CSV format to csv files ready to be included in C source files using #include directive.  It also reindexes tile ids starting from 0 (as in tmx files tile ids start from 1).
Conversion is output to the screen.

Tiled (program)
Create new map options:
CSV  !!

**Comment by lronaldo** [comment]
Dear @Unbedingt,

    This problem was already solved almost 2 years ago, as it can be seen on the closed issue you answered first (#39). Current CPCtelera development version (formerly v1.5) includes the fix. You can download current CPCtelera development branch or either manually apply patches to your version. You can find pathches in these commits: (Dec 16 2015, Jan 13 2016)
25ada380b95c90a7e945307e51fe4e9da37cd5a1
6fe23ecee69292474333a66d849a7924089a92f9
604bae0508bf1e0c802471d0657880e1956ef0d2
e5af6a4733774e047843795defa18b605e60c60c

Hope this helps.
